federator/src/contracts/IAllowTokensV0.ts

```diff
@@ -25,8 +25,8 @@ export class IAllowTokensV0 implements IAllowTokens {
 
   async getConfirmations(): Promise<ConfirmationsReturn> {
     let confirmations = 0; //for rsk regtest and ganache
-    if (this.chainId === 31 || this.chainId === 42) {
-      // rsk testnet and kovan
+    if (this.chainId === 31 || this.chainId === 5) {
+      // rsk testnet and goerli TO DO: ASK AUGUSTO ABOUT CONFIRMATIONS.
```

Comment:
I think we can leave the number of confirmations as it is and remove this comment